app/components/OrganizationMemberList.tsx

```diff
@@ -0,0 +1,161 @@
+import { tw, tx } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import Dropdown from '../icons/TriangleDown'
+import { useState } from 'react'
+import { Checkbox } from './Checkbox'
+import { PageIndicator } from './PageIndicator'
+import { ProfilePicture } from './ProfilePicture'
+
+export enum Role {
+  user,
+  admin,
+}
+
+type OrganizationMemberListTranslation = {
+  edit: string,
+  remove: string,
+  deselectAll: string,
+  selectAll: string,
+  members: string,
+  addMember: string,
+  saveChanges: string,
+  role: string,
+  of: string,
+  roleTypes: (role: Role) => string
+}
+
+const defaultOrganizationMemberListTranslations: Record<Languages, OrganizationMemberListTranslation> = {
+  en: {
+    edit: 'Edit',
+    remove: 'Remove',
+    deselectAll: 'Deselect all',
+    selectAll: 'Select all',
+    members: 'Members',
+    addMember: 'Add member',
+    saveChanges: 'Save changes',
+    role: 'Role',
+    of: 'of',
+    roleTypes: (role: Role) => {
+      return { [Role.admin]: 'Admin', [Role.user]: 'User' }[role]
+    }
+  },
+  de: {
+    edit: 'Bearbeiten',
+    remove: 'Entfernen',
+    deselectAll: 'Auswahl aufheben',
+    selectAll: 'Alle auswählen',
+    members: 'Mitglieder',
+    addMember: 'Miglied hinzufügen',
+    saveChanges: 'Speichern',
+    role: 'Rolle',
+    of: 'von',
+    roleTypes: (role: Role) => {
+      return { [Role.admin]: 'Administrator', [Role.user]: 'Nutzer' }[role]
+    }
+  }
+}
+
+type OrgMember = {
+  email: string,
+  name: string,
+  avatarURL: string,
+  role: Role
+}
+
+type OrganizationDTO = {
+  name: string,
+  members: OrgMember[]
+}
+
+export type OrganizationMemberListProps = {
+  organization: OrganizationDTO,
+  usersPerPage?: number
+}
+
+export const OrganizationMemberList = ({
```

Comment:
I suggest using something like `@tanstack/table` (previously known as `react-table` instead, tables are hard to get right and without a headless component library aiding you in this endeavour it becomes even more difficult to get right

You could argue that it is overkill for such a small table (I would argue against that still) but just for the learning experience (there will be a lot more tables in the future most likely) it is worth to get started with it now